src/CoreFx.Private.TestUtilities/src/System/AssertExtensions.cs

```diff
@@ -317,5 +318,14 @@ public static void Equal(byte[] expected, byte[] actual)
                 throw new AssertActualExpectedException(expectedString, actualString, null);
             }
         }
+
+        /// <summary>Validates that the two sets contains the same elements. XUnit doesn't display the full collections.</summary>
+        public static void Equal<T>(HashSet<T> expected, HashSet<T> actual)
```

Comment:
Don't we need to expose this from the ref?
https://github.com/dotnet/corefx/blob/908cba9d8f275935a9e91b32fd363b5687dc8f86/src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs#L15-L19

Reply:
Yes. Not sure how I managed to build locally. Must have built the wrong flavor.

Reply:
Hah, that's probably why it doesn't build. I suspect neither of us tried building Linux.